### PR TITLE
Improvement translations (ES).

### PR DESCRIPTION
**Pull request by gfvcastro**

### CODE REVIEW

**Comment by acrogenesis** [comment]
@gfvcastro I don't see any changes aside from adding `""` to all strings.

**Comment by gfvcastro** [comment]
@acrogenesis: I order on base `locales/en.yml`, add `will_paginate` translations and update content, example:

* https://github.com/rubygems/rubygems.org/pull/1146/files?diff=unified#diff-526f3087b6dbbf77083d177614d162f6R213
* https://github.com/rubygems/rubygems.org/pull/1146/files#diff-526f3087b6dbbf77083d177614d162f6R176

File: config/locales/es.yml [hunk 1]
```diff
@@ -25,297 +190,122 @@ es:
...
+        edit: "Editar"
+        submit: "Enviar"
       confirmations:
         destroy: "¿Estás seguro?"
       edit:
-        title: Editar aplicación
+        title: "Editar aplicación"
       form:
-        error: Ups! Checa la forma por posible errores
+        error: "Ups! Revisa la forma por posible errores"
```
Comment (acrogenesis) [inline config/locales/es.yml]: :+1:

File: config/locales/es.yml [hunk 2]
```diff
@@ -1,23 +1,188 @@
...
+        handle: "Usuario"
+        password: "Contraseña"
       linkset:
-        bugs: URL del registro de Bugs
-        code: URL del código fuente
-        docs: URL de la documentación
-        mail: URL de la lista de correo
-        wiki: URL del Wiki
+        bugs: "URL del registro de Bugs"
+        code: "URL del código de fuente"
```
Comment (acrogenesis) [inline config/locales/es.yml]: Este me parece que si debe ser `código fuente`. `código de fuente` es como decir source of code. [https://es.wikipedia.org/wiki/Código_fuente](Código fuente) en wikipedia

**Comment by acrogenesis** [comment]
@dwradcliffe Correct me if I'm wrong but I think the standard on yml is that strings that don't need `"` shouldn't have them. If so @gfvcastro should fix it before merge

File: config/locales/es.yml (hunk 2 above)
Comment (gfvcastro) [inline config/locales/es.yml]: @acrogenesis: You are right.

**Comment by dwradcliffe** [comment]
Yes I would prefer no quotes unless they are necessary. I'm evaluating some tooling that might enforce that, so I would rather stick with that rule.

**Comment by gfvcastro** [comment]
@acrogenesis @dwradcliffe: Yes, but other translations use `"`, I thought uniform.

**Pull request closed without merging**